### PR TITLE
Added validation to restrict threshold value to be integer only

### PR DESCRIPTION
**Pull request by paltanmoy**

### CODE REVIEW

File: api/lib/validation/attributeValidator.js [hunk 1]
```diff
@@ -33,6 +33,10 @@ var normalizeEndDate = function(object) {
   return object.end_date;
 }
 
+var isInt = function(n) {
+  return n % 1 === 0;
+}
+
```
Comment (cdlliuy) [inline api/lib/validation/attributeValidator.js]: According to the sample in https://www.npmjs.com/package/jsonschema ,  the change in schemaValidator.js is enough.   Why we need to check attributeValidator again?

File: api/lib/validation/schemaValidator.js [hunk 2]
```diff
@@ -94,7 +94,7 @@ var getScalingRuleSchema = function() {
     'properties' : {
       'metric_type':{ 'type':'string' },
       'breach_duration_secs':{ 'type':'number','minimum': 60,'maximum': 3600 },
-      'threshold':{ 'type':'number'},
+      'threshold':{ 'type':'integer'},
```
Comment (cdlliuy) [inline api/lib/validation/schemaValidator.js]: how about making breach_duration_secs and cool_down_secs to "integer" as well?

**Comment by paltanmoy** [comment]
@cdlliuy build is green after re-try.

**Comment by qibobo** [comment]
LGTM

**Comment by cdlliuy** [comment]
LGTM

**Pull request merged**